Source/DafnyCore/DafnyPrelude.bpl

```diff
@@ -653,164 +653,164 @@ procedure $IterCollectNewObjects(prevHeap: Heap, newHeap: Heap, this: ref, NW: F
 // -- Axiomatization of sets -------------------------------------
 // ---------------------------------------------------------------
 
-type Set T = [T]bool;
+type Set = [Box]bool;
 
-function Set#Card<T>(Set T): int;
-axiom (forall<T> s: Set T :: { Set#Card(s) } 0 <= Set#Card(s));
+function Set#Card(Set): int;
+axiom (forall s: Set :: { Set#Card(s) } 0 <= Set#Card(s));
 
-function Set#Empty<T>(): Set T;
-axiom (forall<T> o: T :: { Set#Empty()[o] } !Set#Empty()[o]);
-axiom (forall<T> s: Set T :: { Set#Card(s) }
+function Set#Empty(): Set;
+axiom (forall o: Box :: { Set#Empty()[o] } !Set#Empty()[o]);
+axiom (forall s: Set :: { Set#Card(s) }
   (Set#Card(s) == 0 <==> s == Set#Empty()) &&
-  (Set#Card(s) != 0 ==> (exists x: T :: s[x])));
+  (Set#Card(s) != 0 ==> (exists x: Box :: s[x])));
 
 // the empty set could be of anything
-//axiom (forall<T> t: Ty :: { $Is(Set#Empty() : [T]bool, TSet(t)) } $Is(Set#Empty() : [T]bool, TSet(t)));
+//axiom (forall t: Ty :: { $Is(Set#Empty() : [Box]bool, TSet(t)) } $Is(Set#Empty() : [Box]bool, TSet(t)));
```

Comment:
Curiousity: I assume this was commented out before because it's useless if T could only ever be Box. It seems even more useless now. Why not remove it entirely?

Reply:
Makes sense. I didn't touch any existing comments, but I can remove this.